hardware/can/Makefile

```diff
@@ -0,0 +1,12 @@
+TOPDIR ?= ../..
+include $(TOPDIR)/.config
+
+$(CAN_SUPPORT)_SRC += hardware/can/can.c \
```

Comment:
Two levels:
1. CAN_SUPPORT
2 a. MCP2515_SUPPORT
2 b. to_be_written_SUPPORT
2 c. ...

Have a look at i2c/master for example.